tests/utils/utils.go

```diff
@@ -697,3 +696,21 @@ func GetThreeSubnets(network interfaces.Network) (
 	Expect(len(subnets)).Should(BeNumerically(">=", 3))
 	return subnets[0], subnets[1], subnets[2]
 }
+
+func CalculateMessageID(
+	teleporterAddress common.Address,
+	sourceBlockchainID ids.ID,
+	destinationBlockchainID ids.ID,
+	messageNonce *big.Int,
+) ids.ID {
+	// TODO: Clean up ABI encoding of parameters.
```

Comment:
double checking on this TODO, wasn't too sure what it was about and if it still applies